src/components/Header/Header.js

```diff
@@ -103,66 +120,30 @@ const Header = ({ logout = () => {}, currentUser = undefined }) => {
     </OfflineHide>
   )
 
-  const handleImageError = () => {
-    setHasImageError(true)
-  }
-
-  const getUserButton = () => {
-    // Avatar with user image
-    if (currentUser && currentUser.picture && !hasImageError) {
-      return (
-        <AvatarWrapper>
-          <CurrentUserImg src={currentUser.picture} alt="" onError={handleImageError} />
-        </AvatarWrapper>
-      )
-    }
-
-    // Avatar with fallback image
-    if (currentUser && currentUser.picture && hasImageError) {
-      return (
-        <AvatarWrapper>
-          <BiggerIconUser />
-        </AvatarWrapper>
-      )
-    }
-
-    // First name
-    if (currentUser && currentUser.first_name) {
-      return (
-        <AvatarWrapper>
-          {currentUser && currentUser.first_name} <IconDown />
-        </AvatarWrapper>
-      )
-    }
-
-    // Full name
-    if (currentUser && currentUser.full_name) {
-      return (
-        <AvatarWrapper>
-          {currentUser && currentUser.full_name} <IconDown />
-        </AvatarWrapper>
-      )
-    }
-
-    // User icon
-    return (
+  const profileImageButton = isProfileImageButtonShowing ? (
+    <AvatarWrapper>
+      <CurrentUserImg src={currentUser?.picture} alt="" onError={handleImageError} />
```

Comment:
The `alt` property should have a value which isn't an empty string

Reply:
Great catch! Thanks!

Reply:
I should add aria label to all the other user icons too if that wasnt previously built